app/decorators/plant_trial_submission_decorator.rb

```diff
@@ -43,7 +43,7 @@ def plant_population
 
   def trait_descriptors
     tds = object.content.step02.trait_descriptor_list.try(:select, &:present?) || []
-    numeric_tds = tds.select { |td| td.to_s.match /\A\d+\z/ }
+    numeric_tds = tds.select { |td| td.to_s.match(/\A\d+\z/) }
```

Comment:
Ha, no free lunch, then... Let's leave it as it is, it's not worth delaying that work because of that.